src/compiler/program.ts

```diff
@@ -921,6 +922,10 @@ namespace ts {
             return sourceFilesFoundSearchingNodeModules.get(file.path);
         }
 
+        function isSourceFileDefaultLibrary(file: SourceFile): boolean {
+            return file.fileName === host.getDefaultLibFileName(options);

```

Comment:
If that the case, we might want to change this method to check for all of the different files. I'll talk to Ron and get his input.